Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -630,14 +636,13 @@ function InvokeDTAExecHostExe([string] $Version, [System.Management.Automation.P
     }
     $exePath = Join-Path -Path $vsRoot -ChildPath $ExeName
     $exePath = "'" + $exePath + "'"
+
     Try
     {
-        $StartDate = New-Object -TypeName DateTime -ArgumentList:(2050,01,01)
-        $FormatHack = ($([System.Globalization.DateTimeFormatInfo]::CurrentInfo.ShortDatePattern) -replace 'M+/', 'MM/') -replace 'd+/', 'dd/'
-        $date1 = $StartDate.ToString($FormatHack)
-
         $session = CreateNewSession -MachineCredential $MachineCredential
-        Invoke-Command -Session $session -ErrorAction Continue -ErrorVariable err -OutVariable out -scriptBlock { schtasks.exe /create /TN:DTAConfig /TR:$args[0] /F /RL:HIGHEST  /SC:ONCE /ST:23:59 ; schtasks.exe /run /TN:DTAConfig;  schtasks.exe /change /disable /TN:DTAConfig } -ArgumentList $exePath
+
+        Write-Verbose -Message "Running schtasks.exe to launch testagent process in the current session" -Verbose
```

Comment:
it would be best to print the exact commandline so that the user can reuse and retry on failure